docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,7 @@ Version history
 
 1.9.0 (pending)
 ===============
+* fault: removed integer percentage support.
```

Comment:
This is already noted in deprecations so I think we can skip this in version history (at least we haven't done this so far). WDYT?

Reply:
I personally feel that we should keep this. It'd serve as a quick reference point to see when a certain feature was removed.

Reply:
OK that's fine with me.